_Week5/group1/README.md

```diff
@@ -0,0 +1,38 @@
+# Group 1 Week 5 Homework Assignment
+## Shaily Acharya, Sylvia Brown, Neel Desai
+## Last updated October 9, 2022
+
+## Overview
+
+This program creates a matrix of one-way ANOVA results and a box plot of the outcome variable over each group variable that was used in the ANOVA test. It is not data-specific; the program is generic and the user can apply any dataset that is built into Stata (i.e., that can be called with `sysuse`).
+
+## Files
+This repository includes: 
+
+- **one_way_anova.do**: The one-way ANOVA program in a .do file.
+- **demonstration.do**: A .do file that demonstrates a variety of uses of the one-way ANOVA program.
+- **README.md**: This README file providing documentation of the program.
+
+## Explanation of Program
+
+Below is an explanation of how to run the program.
+
```

Comment:
I feel this section is self-explanatory given that it is a readme file and removing this chunk will shorten its length

Reply:
Shortened this section in the updated file, thanks for the feedback!